jerry-core/jerry-api.h

```diff
@@ -140,6 +140,13 @@ typedef bool (*jerry_external_handler_t) (const jerry_api_object_t *function_obj
  */
 typedef void (*jerry_object_free_callback_t) (const uintptr_t native_p);
 
+/**
+ * function type applied for each fields in objects
+ */
+typedef bool (*jerry_object_field_foreach_t) (const jerry_api_string_t * field_name_p,
+                                              const jerry_api_value_t * field_value_p,
```

Comment:
Minor formatting:
```js
const jerry_api_string_t *field_name_p,
const jerry_api_string_t *field_value_p,
```

That is the `*` should not have a space before the variable name.